src/common/transformations/include/ngraph_ops/nms_ie_internal.hpp

```diff
@@ -32,6 +32,28 @@ class TRANSFORMATIONS_API NonMaxSuppressionIEInternal : public Op {
                                 const ngraph::element::Type& output_type = ngraph::element::i64,
                                 const ngraph::element::Type& score_output_type = ngraph::element::f32);
 
+    NonMaxSuppressionIEInternal(const Output<Node>& boxes,
+                                const Output<Node>& scores,
+                                const Output<Node>& max_output_boxes_per_class,
+                                const Output<Node>& iou_threshold,
+                                const Output<Node>& score_threshold,
+                                int center_point_box,
+                                bool sort_result_descending,
+                                bool soft_nms,
```

Comment:
soft_nms is always false if input number is less than 6. See [spec](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/sort/NonMaxSuppression_9.md) for input soft_nms_sigma description. it's maybe not needed.

Reply:
done - changes reverted in ngraph